dynamic_instrumentation/src/point/cast.rs

```diff
@@ -76,7 +73,6 @@ pub fn cast_ptr_to_usize<'tcx>(
         // create a raw ptr with `addr_of!`.
         InstrumentationArg::AddrOf(arg) => {
             let arg_place = arg.place().expect("Can't get the address of a constant");
-            let arg_place = remove_outer_deref(arg_place, tcx);
```

Comment:
This appears to have been changed in a merge commit (c32d9b67fa9728b244c1d290266afee41959f9dd). Why?

Reply:
it was a mistake to include it in that commit, that removal was fixing a bug